conf/backup-with-borg.j2

```diff
@@ -30,7 +30,14 @@ fi
 # Backup all apps independently
 apps=$(yunohost app setting {{ app }} apps)
 for app in $(yunohost app list --installed -b | grep id: | cut -d: -f2); do
```

Comment:
The parameters --installed -b should be dropped in the latest version of Yunohost

Reply:
c.f. #43